pallets/stateful-storage/src/benchmarking.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(clippy::unwrap_used, clippy::expect_used)]
```

Comment:
Perhaps all benchmarking.rs files should use this allow statement for both unwrap and expect.

Reply:
I was trying to limit the scope of the lint exceptions.